app/assets/javascripts/discourse/app/components/topic-list.js

```diff
@@ -10,12 +10,14 @@ import TopicBulkActions from "./modal/topic-bulk-actions";
 export default Component.extend(LoadMore, {
```

Comment:
I imagine we will eventually want to update the implementation of `<TopicList>` to Glimmer/gjs. IMO that might be good to do as part of this same feature flag, so that we 'rip the band aid off' all in one go. So perhaps we should create a new modernized version of `TopicList` as part of this PR?

What do you think?

Reply:
"there be (more) dragons" 😅

…but I'm working on it 🙂